src/containers/commitments/index.tsx

```diff
@@ -98,11 +98,11 @@ function Commitments() {
           <Events />
         </div>
 
-        {/* <div className="red section">
+        <div className="red section">
           <IllustratedText imageSrc={trombiImageSrc} alignment="center" title="L'équipe" titleAlignment="center">
             <p>Voici l'équipe qui se cache derrière Cassiopée qui vous propose l'évenement du Gala UTT</p>
           </IllustratedText>
```

Comment:
Tu peux enlever le IllustratedText vu que l'image prend toute la largeur, nan ? Il ne sert pas à grand chose 🤷
Plutôt mettre qqch comme ça du coup _(teste le style, je n'ai pas du tout vérifié ce que ça donne; et importe le titre)_
```suggestion
          <Title alignment="center">L'équipe</Title>
          <img src={trombiImageSrc} alt="Trombinoscope">
          <p>Voici l'équipe qui se cache derrière Cassiopée qui vous propose l'évenement du Gala UTT</p>
```

Reply:
Alors en fait ça j'avais déjà testé de faire juste avec une div et un title. Mais au final ça fait rajouter quasiment le même CSS dans les deux cas. Donc je me disais que c'était mieux de faire un composant réutilisable 🤷 Et pour les petits icones dans le cas de notre image ça tombe mal mais pour d'autres ça peut être sympa je me disais.